icicle/include/poseidon/constants/constants_template.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#ifndef CURVE_POSEIDON_H
+#define CURVE_POSEIDON_H
+
+namespace poseidon_constants_curve {
```

Comment:
what is that file for?

Reply:
Inheritance from Roman's v2 code.
Will be removed in the future.